internal/services/oracle/oracle_network_anchor_data_source.go

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package oracle
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/zones"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/oracledatabase/2025-09-01/networkanchors"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+)
+
+type NetworkAnchorDataSource struct{}
+
+type NetworkAnchorDataModel struct {
+	Name              string            `tfschema:"name"`
+	ResourceGroupName string            `tfschema:"resource_group_name"`
+	Location          string            `tfschema:"location"`
+	Tags              map[string]string `tfschema:"tags"`
+	Zones             zones.Schema      `tfschema:"zones"`
+
+	ResourceAnchorId  string `tfschema:"resource_anchor_id"`
+	ProvisioningState string `tfschema:"provisioning_state"`
+	VnetId            string `tfschema:"vnet_id"`
+	SubnetId          string `tfschema:"subnet_id"`
+
+	CidrBlock                          string `tfschema:"cidr_block"`
+	OciVcnId                           string `tfschema:"oci_vcn_id"`
+	OciVcnDnsLabel                     string `tfschema:"oci_vcn_dns_label"`
+	OciSubnetId                        string `tfschema:"oci_subnet_id"`
+	OciBackupCidrBlock                 string `tfschema:"oci_backup_cidr_block"`
+	OracleDnsForwardingEndpointEnabled bool   `tfschema:"oracle_dns_forwarding_endpoint_enabled"`
+	OracleDnsListeningEndpointEnabled  bool   `tfschema:"oracle_dns_listening_endpoint_enabled"`
+	OracleToAzureDnsZoneSyncEnabled    bool   `tfschema:"oracle_to_azure_dns_zone_sync_enabled"`
+	DnsForwardingEndpointIpAddress     string `tfschema:"dns_forwarding_endpoint_ip_address"`
+	DnsForwardingRuleUrl               string `tfschema:"dns_forwarding_rule_url"`
+	DnsForwardingEndpointNsgRuleUrl    string `tfschema:"dns_forwarding_endpoint_nsg_rule_url"`
+	DnsListeningEndpointAllowedCidrs   string `tfschema:"dns_listening_endpoint_allowed_cidr"`
```

Comment:
Shouldn't the `tfschema` also be `dns_listening_endpoint_allowed_cidrs` since it accepts multiple CIDRs?
```suggestion
	DnsListeningEndpointAllowedCidrs   string `tfschema:"dns_listening_endpoint_allowed_cidrs"`
```